extensions.json

```diff
@@ -94,6 +94,12 @@
       "id": "asabil.meson",
       "repository": "https://github.com/asabil/vscode-meson"
     },
+    {
+      "id": "asciidoctor.asciidoctor-vscode",
+      "repository": "https://github.com/asciidoctor/asciidoctor-vscode",
```

Comment:
Please remove the `repository` property. It is not necessary when we have a download URL.

Reply:
Sorry. Commit done.